### PR TITLE
USD Integration

### PR DESCRIPTION
**Pull request by abhihjoshi**
Three files:
- mujoco/python/mujoco/usd_exporter.py
- mujoco/python/mujoco/usd_component.py
- mujoco/python/mujoco/usd_utils.py

The USD exporter class handles all the overall USD file and materials. The USD component files handles all the objects in the scene.

### CODE REVIEW

File: python/dist/mujoco-3.0.2.tar.gz
Comment (nimrod-gileadi) [inline python/dist/mujoco-3.0.2.tar.gz:1]: Remove from this PR.

**Review by nimrod-gileadi — CHANGES_REQUESTED** [review]
First look.

**Comment by yuvaltassa** [comment]
Let's delete the `usd_` prefix to avoid stuttering. `usd/component.py` etc.

File: python/mujoco/__init__.py [hunk 1]
```diff
@@ -46,6 +46,9 @@
 from mujoco._structs import *
 from mujoco.gl_context import *
 from mujoco.renderer import Renderer
+from mujoco.usd.usd_exporter import USDExporter
```
Comment (nimrod-gileadi) [inline python/mujoco/__init__.py:49]: Remove these imports so the module can be loaded separately.

**Review by nimrod-gileadi — APPROVED** [review]

**Review by crewmatt — APPROVED** [review]

**Review by crewmatt — APPROVED** [review]

**Comment by yuvaltassa** [comment]
This needs to be rebased.
Probably better to just close this PR and start a new one at head? 
The discussion is long and mostly irrelevant...

**Pull request closed without merging**